client/src/main/java/com/battleships/connection/commands/FleetSunk.java

```diff
@@ -0,0 +1,32 @@
+package com.battleships.connection.commands;
+
+import com.battleships.Translator;
+import com.battleships.connection.Connection;
+import javafx.application.Platform;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+
+import java.util.Optional;
+
+public class FleetSunk extends AbstractServerCommand {
+    public FleetSunk(Object value) {
+        super(value);
+    }
+
+    @Override
+    public void execute() {
+        Platform.runLater(() -> showWinnerAlert());
+    }
+
+    private void showWinnerAlert() {
+        Alert alert = new Alert(Alert.AlertType.INFORMATION);
+        alert.setHeaderText(null);
+        alert.titleProperty().bind(Translator.createStringBinding("game_won"));
```

Comment:
Duplication in a file translation.properties:

won=You won!
won_header=You are winner!


game_won=Winner!
game_won_info=You won, congrats!

Reply:
fixed